content/en/docs/Upgrade to PHP 8.1/_index.md

```diff
@@ -0,0 +1,479 @@
+---
+title: "Upgrade to PHP 8.1"
+linkTitle: "Upgrade to PHP 8.1"
+weight: 4
+---
+**# Estimated Upgrade Time: 1.5 Hours**
+
+We attempt to make the shift from PHP 7.2 to PHP 8.1 as simple as possible. Some services have significantly changed due to third-party API suppliers. Below we have listed all services that have experienced noticeable changes. You do not have to update all of the following services, but if you use any of them, be sure to review the appropriate section to start building your dream API sooner.
+
+{{< alert color="warning" title="Note" >}}
+This article is designed for the fact that you do not use our automatic installers. If you have a different operating system or don't want to do everything by yourself, you can find the relevant guide [here](../installing-and-configuring-dreamfactory/#using-the-dreamfactory-installer). These steps are dedicated that you are using the [_Supported Ubuntu Release_](https://wiki.ubuntu.com/Releases).
+{{< /alert >}}
+
+<h2>Switch to PHP 8.1</h2>
+In this section, we will show you how to migrate from PHP 7.4 to PHP 8.1 on Ubuntu 20.04 as an example. 
+
+First, we need to add PPA:
+
+```bash
+sudo add-apt-repository ppa:ondrej/php -y
+```
+
+Then install PHP 8.1 and the required dependencies:
+
+```bash
+sudo apt-get install -y php8.1-common \
+php8.1-xml \
+php8.1-cli \
+php8.1-fpm \
+php8.1-curl \
+php8.1-mysqlnd \
+php8.1-sqlite \
+php8.1-soap \
+php8.1-mbstring \
+php8.1-zip \
+php8.1-bcmath \
+php8.1-dev \
+php8.1-ldap \
+php8.1-pgsql \
+php8.1-interbase \
+php8.1-gd \
+php8.1-sybase
+```
+
+After successful installation we need to manually switch to PHP 8.1:
+
+```bash
+sudo update-alternatives --set php /usr/bin/php8.1 \
+sudo update-alternatives --set phar /usr/bin/phar8.1 \
+sudo update-alternatives --set phar.phar /usr/bin/phar.phar8.1 \
+```
+
+Then select the version of PHP you need by using:
+
+```bash
+sudo update-alternatives --config php
+```
+
+{{< alert color="warning" title="Info" >}}
+To learn how to install dependencies for the DreamFactory packages that you are interested in, you may use the table of contents
+below.
+
+Command `pecl install` may fail with message like:
+
+`pecl/sqlsrv is already installed and is the same as the released version 5.10.1 install failed`
+
+In this case use force install: `pecl install -f [extension name]`
```

Comment:
```suggestion
In this case use force install: `pecl install -f <extension name>`
```

It 's common for CLI utils to use `[]` for optional arguments and `<>` for required one